view/src/components/produce.js

```diff
@@ -0,0 +1,713 @@
+              </Grid>
+              {this.state.produceObjects.map((produce) => (
+                <Grid item xs={12}>
+                  <Card className={classes.root} variant="outlined">
+                    <CardContent>
+                      <Typography variant="h5" component="h2">
+                        {produce.name}
+                      </Typography>
+                      <Box
+                        display="flex"
+                        flexDirection="row"
+                        flexWrap="wrap"
+                        p={0}
+                        m={0}
+                      >
+                        <Box p={3}>
+                          <Typography
+                            className={classes.pos}
+                            color="textSecondary"
+                          >
+                            Shipping Temperatures in Reefer (°F):
+                          </Typography>
+                          <Typography variant="body2" component="p">
+                            Maintenance Temperature:{" "}
+                            {produce.shippingMaintenanceTemperatureLow} -{" "}
+                            {produce.shippingMaintenanceTemperatureHigh}
+                            <br />
+                            Preset Temperature:{" "}
+                            {produce.shippingPresetTemperature}
+                          </Typography>
+                        </Box>
+                        <Box p={3}>
+                          <Typography
+                            className={classes.pos}
+                            color="textSecondary"
+                          >
+                            Pricing (in USD / lb):
+                          </Typography>
+                          <Typography variant="body2" component="p">
+                            USDA Price: ${produce.price}
+                            <br />
+                            Average Price Paid by Farmlink: ${produce.pricePaid}
+                          </Typography>
+                        </Box>
+                        <Box p={3}>
+                          <Typography
+                            className={classes.pos}
+                            color="textSecondary"
+                          >
+                            Internal Statistics:
+                          </Typography>
+                          <Typography variant="body2" component="p">
+                            Amount Moved by FarmLink (lbs):{" "}
+                            {produce.amountMoved}
+                          </Typography>
+                        </Box>
+                      </Box>
+                    </CardContent>
+                    <CardActions>
+                      <Button
+                        size="small"
+                        color="primary"
+                        onClick={() => this.handleViewOpen({ produce })}
+                      >
+                        {" "}
+                        View{" "}
```

Comment:
I realized why all these spaces were popping up. They are being added by prettier. As a result, I am assuming they are part of some style norm. Thoughts?